apps/music/js/ui/views/list_view.js

```diff
@@ -432,6 +433,14 @@ var ListView = {
 
         break;
 
+      case 'keypress':
+        if (target.id === 'views-list-search-input') {
+          if (evt.keyCode == 13) { // VK_Return
```

Comment:
Probably we can use `KeyboardEvent.DOM_VK_RETURN` which is the same as the keyCode 13, and no need to add comment.